save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -276,6 +275,7 @@ class DownloadFilesController(
             when (type) {
                 AvatarType.ORGANIZATION -> organizationService.saveAvatar(owner, it)
                 AvatarType.USER -> userDetailsService.saveAvatar(owner, it)
+                else -> throw IllegalStateException("Not supported type: $type")
```

Comment:
Better replace with NONE here. The whole `when`-expression will remain exhaustive, but we'll receive a compilation error immediately after we add a new avatar type (failing fast is good).

Reply:
@Cheshiriks, why we need NONE at all? it's invalid case and we don't need to support. Since `kotlin` supports validating `when` on compilation -- better to remove NONE and we will get `compilation` error when a new type of avatar will be added (as @0x6675636b796f75676974687562 suggested)

Reply:
Agree. Deleted it.